lib/streamlit/runtime/caching/hashing.py

```diff
@@ -101,7 +106,7 @@ def _get_error_message_args(
         self,
         orig_exc: BaseException,
         failed_obj: Any,
-    ) -> Dict[str, Any]:
+    ) -> dict[str, Any]:
```

Comment:
Is this necessary?

Reply:
these are the new/future annotations. The upper case versions are the old way of typing these structures. Vincent already started adding that in a couple of modules and I have been preparing a bigger PR to update that across our codebase here: https://github.com/streamlit/streamlit/pulls